src/System.Drawing.Common/tests/Drawing2D/MatrixTests.cs

```diff
@@ -203,46 +208,33 @@ public static IEnumerable<object[]> Equals_TestData()
         [MemberData(nameof(Equals_TestData))]
         public void Equals_Other_ReturnsExpected(Matrix matrix, object other, bool expected)
         {
-            try
+            using (matrix)
+            using (other as IDisposable)
             {
                 Assert.Equal(expected, matrix.Equals(other));
                 if (other is Matrix otherMatrix)
                 {
                     Assert.Equal(ReferenceEquals(matrix, other), matrix.GetHashCode().Equals(other.GetHashCode()));
                 }
             }
-            finally
-            {
-                matrix.Dispose();
-                (other as IDisposable)?.Dispose();
-            }
         }
 
         [ConditionalFact(Helpers.IsDrawingSupported)]
         public void Equals_Disposed_ThrowsArgumentException()
         {
-            var matrix = new Matrix();
-            matrix.Dispose();
-
-            AssertExtensions.Throws<ArgumentException>(null, () => matrix.Equals(new Matrix()));
+            AssertExtensions.Throws<ArgumentException>(null, () => s_disposedMatrix.Equals(new Matrix()));
```

Comment:
Why is using the cached static better than creating and disposing it each time?  There's arguably a benefit to the latter, which is that we're then not subject to any mutations accessing one of these methods after Dispose may do and the resulting non-determinism that would come from test evaluation order.  I'd be inclined to revert this and just do the ctor+Dispose in every test as it was doing, unless there's a really good reason not to.

Reply:
Fair enough, I converted the static field to a function which returns a disposed region/matrix.